dom/src/main/scala/org/http4s/dom/WebSocketClient.scala

```diff
@@ -67,15 +68,17 @@ object WebSocketClient {
             ws.binaryType = "arraybuffer" // the default is blob
 
```

Comment:
Perhaps we can leave a comment here explaining why we choose not to handle `error` events at all. If I understand correctly, then the reason we can do so is because, according to the spec,
1) an `error` event is *always* followed by a `close` event and
2) an `error` event doesn't carry any useful information *by design*

Reply:
Well put, let's add it :)